components/counter/Counter.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+import theme from './theme.css';
+
+class Counter extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    color: PropTypes.oneOf([ 'neutral', 'ruby' ]),
+    size: PropTypes.oneOf([ 'small', 'medium' ]),
+  };
+
+  static defaultProps = {
+    color: 'neutral',
+    size: 'medium',
+  };
+
+  render () {
+    const {
+      children,
+      className,
+      color,
+      size,
+      ...others
+    } = this.props;
+
+    const classes = cx(
+      theme.counter,
+      theme.monospaced,
+      theme[color],
+      theme[size],
+      {
+        [theme.circular]: !children,
+        [theme.rounded]: children,
```

Comment:
I would just make the css rounded by default, and only add the `circular` "variant" when necessary. This way you can drop the rounded class, and you end up with just circular being a variant.